src/pages/EnablePayments/AdditionalDetailsStep.js

```diff
@@ -327,18 +369,18 @@ class AdditionalDetailsStep extends React.Component {
                                 <DatePicker
                                     containerStyles={[styles.mt4]}
                                     label={this.props.translate(this.fieldNameTranslationKeys.dob)}
-                                    onInputChange={val => this.clearErrorAndSetValue('dob', val)}
+                                    onInputChange={val => this.clearDateErrorsAndSetValue(val)}
```

Comment:
Were you able to test this on `iOS Safari`?  I get a different date format and unable to move forward.

<img width="435" alt="image" src="https://user-images.githubusercontent.com/3069065/177059148-ec9b2cd8-fb79-46fc-b46f-0cf88662d880.png">

Reply:
I just tested this again and it shows `Must be over 18`